### PR TITLE
[feat]: Adding css for buttons

### PR DESCRIPTION
**Pull request by raghav2110**
Hi, I have added css for button component.
I have 4 different classes for:
1. Primary Button
2. Secondary Button
3. Icon Button
4. Floating Action Button

Please review my code for same
Screenshot 2022-01-29 at 11 33 53 AM
.

### CODE REVIEW

**Comment by aryan-13** [comment]
The code looks good!

File: components/buttons/buttons.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+.btn-primary:hover{
+    background-color: #1a9fc7;
+}
+
+.btn-primary{
+    color: whitesmoke;
+}
+
+.btn-secondary{
+    color: #1FABD7;
```
Comment (lannisterrr) [inline components/buttons/buttons.css:23]: colors can go to a variables

File: components/buttons/buttons.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
+.btn{
+    width: max-content; 
+    letter-spacing: 1px;
+    background-color: #1FABD7;
+    padding: 0.8rem 0.8rem;
```
Comment (lannisterrr) [inline components/buttons/buttons.css:5]: padding can go to seperate files as variables.

File: components/buttons/buttons.css (hunk 1 above)
Comment (raghav2110) [reply components/buttons/buttons.css:23]: Hi siddhant , I have not created base file yet. Once the components are done will replace the colour with variable. Will this approach work.

File: components/buttons/buttons.css (hunk 2 above)
Comment (raghav2110) [reply components/buttons/buttons.css:5]: Okay 👍🏻

**Review by ZuberDunge — COMMENTED** [review]
Excellent code structure and great naming practices followed. Good to go.

**Pull request merged**